corehq/apps/userreports/tasks.py

```diff
@@ -11,10 +11,13 @@
 from dimagi.utils.couch.database import iter_docs
 from dimagi.utils.couch.cache.cache_core import get_redis_client
 
+from corehq.util.couch_helpers import paginate_view
 from corehq.apps.domain.dbaccessors import get_doc_ids_in_domain_by_type
 from corehq.apps.userreports.models import DataSourceConfiguration, StaticDataSourceConfiguration
 from corehq.apps.userreports.sql import IndicatorSqlAdapter
 
+CHUNK_SIZE = 1000
```

Comment:
for just streaming the IDs I think this is a very conservative chunk size (10,000 or more probably totally fine)